boa/contracts/vyper/vyper_contract.py

```diff
@@ -467,6 +476,20 @@ def __repr__(self):
         return repr(self.dump())
 
 
+# data structure to represent the constants in a contract
+class ConstantsModel:
+    def __init__(self, compiler_data: CompilerData):
+        for k, v in compiler_data.vyper_module_unfolded._metadata["namespace"].items():
```

Comment:
@charles-cooper is there an easier way to get all constants? Is this going to work every time? :thinking:

Reply:
i think we should work on this off the 0.4.0 branch, since the way constants are handled changed, and we are about to upgrade to 0.4.0 anyways